### PR TITLE
[Selenium] Add GitSelfSignCert typescript test to 7.32.x branch

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
Add GitSelfSignCert typescript test to 7.32.x branch to start on CRW 2.10.0.

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
@SkorikSergey : is there an issue this PR is fixing?

**Review by dmytro-ndp — APPROVED** [review]
looks good to me

File: tests/e2e/tests/e2e/GitSelfSignCert.spec.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+ const dashboard: Dashboard = e2eContainer.get(CLASSES.Dashboard);
+ const browserTabsUtil: BrowserTabsUtil = e2eContainer.get(CLASSES.BrowserTabsUtil);
+ const editor: Editor = e2eContainer.get(CLASSES.Editor);
+
+ const workspaceName = 'gitSelfSignCert';
+
+
+ suite('Checking git + self sign cert', async () => {
+     const workspacePrefixUrl: string = `${TestConstants.TS_SELENIUM_BASE_URL}/dashboard/#/ide/${TestConstants.TS_SELENIUM_USERNAME}/`;
+     const gitProjectUrl = 'https://10.0.104.86:10080/maxura/gitService.git';
```
Comment (dmytro-ndp) [inline tests/e2e/tests/e2e/GitSelfSignCert.spec.ts]: IMHO `gitProjectUrl` would better to be configurable with dedicated constant, having default value `'https://10.0.104.86:10080/maxura/gitService.git'`

File: tests/e2e/tests/e2e/GitSelfSignCert.spec.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        await projectTree.expandPathAndOpenFile('gitService', committedFile);
+        await editor.type(committedFile, currentDate + '\n', 1);
+        await gitPlugin.openGitPluginContainer();
+        await gitPlugin.waitChangedFileInChagesList(committedFile);
+        await gitPlugin.stageAllChanges(committedFile);
+        await gitPlugin.waitChangedFileInChagesList(committedFile);
+        await gitPlugin.typeCommitMessage(this.test!.title + currentDate);
+        await gitPlugin.commitFromCommandMenu();
+        await gitPlugin.pushChangesFromCommandMenu();
+        await quickOpenContainer.typeAndSelectSuggestion('git-admin', 'Git: https://10.0.104.86:10080 (Press \'Enter\' to confirm your input or \'Escape\' to cancel)');
```
Comment (dmytro-ndp) [inline tests/e2e/tests/e2e/GitSelfSignCert.spec.ts]: The same is here https://github.com/eclipse/che/pull/20091/files#r665233174

File: tests/e2e/tests/e2e/GitSelfSignCert.spec.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        await quickOpenContainer.typeAndSelectSuggestion('admin', 'Git: https://git-admin@10.0.104.86:10080 (Press \'Enter\' to confirm your input or \'Escape\' to cancel)');
```
Comment (dmytro-ndp) [inline tests/e2e/tests/e2e/GitSelfSignCert.spec.ts]: the same is here https://github.com/eclipse/che/pull/20091/files#r665233174

**Comment by SkorikSergey** [comment]
@dmytro-ndp Thank you for review. There is added workaround for https://github.com/eclipse/che/issues/19793 issue.

**Pull request merged**